### ISSUE TITLE
Short option `nf` did not work

### ISSUE THREAD

**Issue by llaville** [overtrue/phplint#167]
Affect all current stable versions v3.2.0, v4.3.0, v5.3.0, v6.0.0

When Invoking lint command with `--nf` short option, we got unexpected results : 

```
  [Symfony\Component\Console\Exception\RuntimeException]
  The "-f" option does not exist.

```

Reason : Symfony console check for `n` option (that exists => `-n, --no-interaction`) then try with `f` option that is not declared.

**Comment by llaville** [comment]
@overtrue Do you want to keep short option ? In this condition what is your suggestion ?
Or could we consider to remove short option and keep only the long option (for a very quick fix)

**Comment by overtrue** [comment]
Please remove them, Thanks.

**Comment by llaville** [comment]
Available in latest releases 3.3.1, 4.4.0, 5.4.1 and 6.0.3
